apiary.apib

```diff
@@ -979,6 +979,130 @@ Outputs the list of peers that were removed.
 
 # Group dns
 
+DNS link resolver.
+
+## dns [GET /dns{?arg}]
+Multihashes are hard to remember, but domain names are usually easy to
+remember. To create memorable aliases for multihashes, DNS TXT
+records can point to other DNS links, IPFS objects, IPNS keys, etc.
+This command resolves those links to the referenced object.
+
+The following response expamles assume this DNS TXT record:
+
+```
+ipfs.io. TXT "dnslink=/ipfs/QmRzTuh2Lpuz7Gr39stNr6mTFdqAghsZec1JoUnfySUzcy ..."
```

Comment:
It doesn't do harm, but I'd remove it. With CNAME records the record value has to end in `.` though :) E.g. `foobar.com CNAME gateway.ipfs.io.`.